draft-vvv-webtransport-quic.md

```diff
@@ -218,17 +226,13 @@ than once.
 ### Origin Field
 
 In order to allow the server to enforce its origin policy, the user agent has to
-communicate the origin in the client indication.  This can be accomplished using
-the "Origin" field:
+communicate the origin in the handshake.  This can be accomplished using the
+"Origin" field:
 
 Name:
 
 : Origin
```

Comment:
Shouldn't this be lower-cased?

Reply:
I'm following the HTTP convention here where headers look like "Content-Type", even though on the wire, HTTP/2 and HTTP/3 encode those as all-lowercase.